open-api/README.md

```diff
@@ -0,0 +1,23 @@
+# Open API spec
+
+The `rest-catalog-open-api.yaml` defines the REST catalog interface.
+
+## Lint
+
+To make sure that the open-api definition is valid, you can run the `lint` command:
+
+```sh
+make install
+make lint
+```
+
+## Generate Python code
+
+When reviewing changes in the spec, it helps to see what kind of code is being generated from the spec. We generate Python code to make this process easier. Before committing, make sure to run:
```

Comment:
I'm guessing you mean before committing changes to the Spec with `Before committing`? Maybe we can simplify/change this sentence to:


*When updating `rest-catalog-open-api.yaml`, make sure to update `rest-catalog-open-api.py` with the spec changes by running the following commands:*
```sh
make install
make generate
```